### PR TITLE
[API-188] CLI sample app re-added

### PR DESCRIPTION
**Pull request by betacar**
Picks up after #62 was reverted.

### CODE REVIEW

**Review by githuib — COMMENTED** [review]
Do we want to go for the CLI approach in the sample apps, instead of just providing an interface for the user where they can pass arguments like the base URL, client ID etc.?

This way they would have the freedom of loading the parameters from wherever they want to (config file, CLI args, hardcoded, or anything else), and saves us from adding a dependency for CLI stuff (like yargs in this case).

**Comment by betacar** [comment]
@githuib That's a question for @bugra-okcu.

**Comment by githuib** [comment]
Yeah wanted to ask on slack, but of course that went offline when I just typed a whole message..

File: samples/cli.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  })
+  .then((data) => {
+    console.log(data)
+    const file = `${__dirname}/testasset.png`;
+    return bynder.uploadFile({
+      filename: path.basename(file),
+      body: fs.readFileSync(file),
+      data: {
+        brandId: brand.id,
+        name: 'test asset new version',
```
Comment (githuib) [inline samples/cli.js:33]: No need to pass brandId and name when uploading a new version.

File: package.json [hunk 2]
```diff
@@ -35,7 +35,8 @@
     "eslint-plugin-import": "^2.22.0",
     "jest": "^24.9.0",
     "jsdoc": "^3.6.6",
-    "rollup": "^2.28.2"
+    "rollup": "^2.28.2",
+    "yargs": "^16.2.0"
```
Comment (githuib) [inline package.json:39]: Is this still used?

**Review by githuib — COMMENTED** [review]
I wonder if it actually works in practice to do the new version upload immediately after the initial upload. Did you test this on stage / prod?

**Comment by bugra-okcu** [comment]
@githuib  yes, I've tested and it works. I'll check with Jake though, to confirm immediate new version upload is expected.

File: package-lock.json [hunk 3]
```diff
@@ -5859,6 +5857,26 @@
             "prompts": "^2.0.1",
             "realpath-native": "^1.1.0",
             "yargs": "^13.3.0"
+          },
+          "dependencies": {
+            "yargs": {
```
Comment (betacar) [inline package-lock.json:5862]: When you update a dependency on `package.json`, you should re-run `npm i` so the lock file is also up-to-date.

**Review by betacar — COMMENTED** [review]
@bugra-okcu Please re-run `npm i` so `package-lock.json` is aligned with `package.json`.

**Review by githuib — APPROVED** [review]

**Pull request merged**